src/quo2/components/numbered_keyboard/keyboard_key/component_spec.cljs

```diff
@@ -0,0 +1,42 @@
+(ns quo2.components.numbered-keyboard.keyboard-key.component-spec
```

Comment:
Btw these are component tests, not e2e tests :) e2e tests are handled by the QA team